Source/WebCore/rendering/style/StyleRareNonInheritedData.h

```diff
@@ -252,6 +252,7 @@ class StyleRareNonInheritedData : public RefCounted<StyleRareNonInheritedData> {
     LengthPoint offsetPosition;
     LengthPoint offsetAnchor;
     OffsetRotation offsetRotate;
+    unsigned overflowAnchor : 1; // Scroll Anchoring- Overflow Anchor
```

Comment:
More this up next to the other bitfield entries to pack things better.

Reply:
Fixed.